src/main/java/controller/PassengersController.java

```diff
@@ -0,0 +1,125 @@
+package controller;
+
+import db.DatabaseHandler;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Timestamp;
+
+public class PassengersController extends DatabaseHandler  { // no need to extend // nobody is using entities but nobody asked about this
```

Comment:
is this also your part? I thought Oleksandr was creating PassengerController :D

Reply:
Yes, I checked and tested his code and forgot to erase it from my branch